src/site/content/en/blog/rebelmouse/index.md

```diff
@@ -0,0 +1,216 @@
+---
+layout: post
+title: How Rebelmouse grew loyalty and engagement by optimizing Core Web Vitals
+subhead: |
+  A case study measuring the full impact and benefits of high performance scores on Core Web Vitals
+description: |
+  A case study measuring the full impact and benefits of high performance scores on Core Web Vitals
+authors:
+  - andreabreanna
+date: 2022-04-14
+alt: The RebelMouse mascot set against a blue backdrop. The mascot is a grey mouse with a red kerchief on its head, and is standing in front of a pride flag.
+hero: image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/Y0KflFCo0GclHd9JK8iE.png
+thumbnail: image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/PVMV1zKjJs4MsJRFIzJs.png
+tags:
+  - blog
+  - web-vitals
+  - case-study
+---
+
+[RebelMouse is a cloud-based CMS](https://www.rebelmouse.com/) with the data-driven tools and creative strategy publishers need to provide the best experience possible for audiences, both new and old. We power high-traffic web properties for media companies like Upworthy, Narcity, Raw Story, Brit + Co, and The Independent, as well as high-value sites like Penske Trucks, Vault12, Gaingels, and many others.
+
+Our platform reaches over 100 million people monthly in the U.S. alone. The company was founded on the belief that the open web needs to perform at the highest levels of consumer expectations to thrive, and the standards set by Google's Core Web Vitals align with this user-first vision.
+
+For more than a decade, our global team has been engineering a CMS that creates positive and intuitive user experiences that translate into meaningful business. Core Web Vitals are a benchmark that ensure this symbiotic loop continues across every new site we power.
+
+## Why prioritizing performance means better business
+
+Many publishers doubt whether excellent scores on Core Web Vitals are worth the effort. They also worry that even a quality user experience won't deliver the boost in readership they need. Fortunately, our prioritization of performance at RebelMouse has given us the data to prove that Core Web Vitals are an excellent way to generate new traffic and build the user loyalty needed to have a successful business.
+
+In order to illustrate this, we analyzed the audience behavior of a number of sites we power, both before and after we optimized for high performance scores on Core Web Vitals. In particular, we defined types of users to separate the audiences into the following key categories:
+
+- **New Visitors:** Users who visited a site for the first time.
+- **Returning Visitors:** Users who had at least two (2) sessions.
```

Comment:
I don't think we need the numbers in brackets here. Style guide is to spell out the word.